db/go_level_db.go

```diff
@@ -207,10 +208,13 @@ type goLevelDBIterator struct {
 var _ Iterator = (*goLevelDBIterator)(nil)
 
 func newGoLevelDBIterator(source iterator.Iterator, start, end []byte, isReverse bool) *goLevelDBIterator {
-	if isReverse {
-		panic("not implemented yet") // XXX
+	if isReverse && start == nil {
+		for !source.Last() {
```

Comment:
This doesn't seem right.... I'll submit a fix to this branch.  https://github.com/syndtr/goleveldb/blob/master/leveldb/iterator/iter.go#L29

Reply:
Don't know why I thought Last was just checking and not seeking... Thanks for catching that.